core/src/test/scala/integration/kafka/admin/TopicCommandIntegrationTest.scala

```diff
@@ -758,21 +731,15 @@ class TopicCommandIntegrationTest extends KafkaServerTestHarness with Logging wi
     val notUnderMinIsrTopic = "not-under-min-isr-topic"
     val offlineTopic = "offline-topic"
     val fullyReplicatedTopic = "fully-replicated-topic"
+    val replicationFactor: Short = 6
```

Comment:
This can be further moved into a test class level constant as it represents the number of brokers in the cluster that are started in the setup() for this test. see: 
https://github.com/apache/kafka/blob/48eb8c90ef5f474e54694b74377d20f9e378174c/core/src/test/scala/integration/kafka/admin/TopicCommandIntegrationTest.scala#L57

Reply:
Thanks for your explanation, I'm done.